src/PrestaShopBundle/Resources/config/services/adapter/kpi.yml

```diff
@@ -54,3 +54,31 @@ services:
             - '@translator'
             - '@prestashop.adapter.legacy.kpi_configuration'
             - '@=service("prestashop.adapter.legacy.context").getAdminLink("AdminStats", true, {"ajax": 1, "action": "getKpi", "kpi": "products_per_category"})'
+
+    prestashop.adapter.kpi.most_common_customers_gender:
+        class: 'PrestaShop\PrestaShop\Adapter\Kpi\MostCommonCustomersGenderKpi'
+        arguments:
+            - '@translator'
+            - '@prestashop.adapter.legacy.kpi_configuration'
+            - '@=service("prestashop.adapter.legacy.context").getAdminLink("AdminStats", true, {"ajax": 1, "action": "getKpi", "kpi": "customer_main_gender"})'
```

Comment:
are you sure about these arguments, you put the arguments in `$sfRouteParams` but for legacy links you need to use `$params` so I might be wrong but I think the correct syntax would be:

```
    - '@=service("prestashop.adapter.legacy.context").getAdminLink("AdminStats", true, {}, {"ajax": 1, "action": "getKpi", "kpi": "customer_main_gender"})'
```

Reply:
actually, current argument order works well, are you sure it wouldnt break? :thinking:

Reply:
oh right, I understood why
That's because the method `getAdminLink` from `LegacyContext` uses the same argument for both third and forth argument in `LinkCore::getAdminLink`
My bad!